src/base/dev/misc/timers.c

```diff
@@ -439,7 +439,7 @@ void pit_control_outp(ioport_t port, Bit8u val)
   switch (latch) {
     case 2:
       if (config.speaker == SPKR_NATIVE) {
-        safe_port_out_byte(0x43, val);
+        port_outb(0x43, val);
```

Comment:
you'll need `std_port_outb` here, as this is already in the port handler for 0x43, otherwise you're looping infinitely unless I missed something.